.circleci/config.yml

```diff
@@ -2,34 +2,24 @@ version: 2
 
 references:
   container_config: &container_config
-    setup_remote_docker:
-      docker_layer_caching: true
     docker:
-      - image: aeternity/builder
-        user: builder
+      - image: aeternity/infrastructure:master
```

Comment:
I admit I did not get my head around this PR fully though I would expect corresponding PR removing from builder Docker image [packer and terraform](https://github.com/aeternity/docker-builder/blob/ec4a8d4eb3c19b98319a16dd2d541f734c599f9a/Dockerfile#L32-L40), to be merged shortly after this?

Reply:
Yes.